.ci/generate_bear_requirements.py

```diff
@@ -126,8 +126,9 @@ def write_gem_requirements(requirements):
     for requirement in requirements:
         gem_dependencies.append(
             {'name': requirement.package,
-             'version': requirement.version,
-             'required': requirement.require})
```

Comment:
Also, the template file `Gemfile.jinja2` needs to be updated if the `required` key is not used.